e2e/tests/workspace/workspace-edit.spec.ts

```diff
@@ -22,7 +22,7 @@ describe('Editing workspace via workspace card snowman menu', () => {
    * - Verify Workspace Information in ABOUT tab.
    */
   test('User as OWNER can edit workspace', async () => {
-    const workspaceCard = await findOrCreateWorkspaceCard(page);
+    const workspaceCard = await findOrCreateWorkspaceCard(page, { workspaceName });
```

Comment:
Fixed bug. ` { workspacename }` should be specified. This test reuse same workspace for all tests. If not exist, it will be created.